app/models/manageiq/providers/cloud_manager/vm.rb

```diff
@@ -58,6 +58,10 @@ def validate_timeline
     {:available => true, :message => nil}
   end
 
+  def cloud_volumes
+    hardware.disks.select(&:backing).map(&:backing)
```

Comment:
I think the CI might be failing because of this? My guess would be that this relation is not defined in the mocked data

Could you base you PR on this https://github.com/ManageIQ/manageiq/pull/8143 ? I am putting the proper VM <-> cloud_volumes relations

So that means you can delete this cloud_volumes method.